frontend/src/components/HeaderDropdown/HeaderDropdown.js

```diff
@@ -40,6 +46,7 @@ const HeaderDropdown = ({ history, user, setJwt }) => {
         <Link className="header-dropdown-item" to="/profile#settings">
           <DropdownItem className="header-dropdown-item">Settings</DropdownItem>
         </Link>
+        {user.is_admin === true ? showAdminLink() : null}
```

Comment:
You could just use `user.is_admin` here. It's already boolean, so the `=== true` is redundant.

Same on the other page.